include/custom_op/custom_op_lite.h

```diff
@@ -458,16 +458,16 @@ class OrtGraphCudaKernelContext : public CUDAKernelContext {
   static const int cuda_resource_ver = 1;
 
   OrtGraphCudaKernelContext(const OrtApi& ort_api, const OrtKernelContext& ctx) : api_(ort_api) {
-    api_.KernelContext_GetResource(&ctx, cuda_resource_ver, CudaResource::cuda_handle_t, &cuda_stream_);
+    OrtStatusPtr result = api_.KernelContext_GetResource(&ctx, cuda_resource_ver, CudaResource::cuda_handle_t, &cuda_stream_);
     if (!cuda_stream_) {
```

Comment:
should we also check the result is not nullptr here to avoid a potential unused the variable warning?

Reply:
Yeah sure. I also added the check for `resource != nullptr`, just in case.